code/Player/MainPlayer.cs

```diff
@@ -21,15 +22,18 @@ public MainPlayer()
         {
             if (Host.IsServer)
             {
-                Inventory = new Inventory(MaxInventoryCapacity);
+                Inventory = new Inventory(MaxInventoryCapacity)
+                {
+                    Owner = this
```

Comment:
Why owner stuf f ?

Reply:
It's better to define who owns the inventory so we can tell the network system "Send the inventory only to its owner".

Reply:
Owner is facepunch stuff ?

Reply:
Yep every entity has an Owner